### PR TITLE
fix: Add Jaideep Guntupalli to the list

### LINKED ISSUE #853: Jaideep Guntupalli
Jaideep_Readme

### PR DESCRIPTION
**Pull request by JaideepGuntupalli**
resolve #853

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -1618,5 +1618,11 @@
     "githubUsername": "krishguptadev",
     "imageUrl": "https://user-images.githubusercontent.com/91655303/142618933-d2dead94-c735-47ac-92d8-025c23a2c07d.png",
     "issueNumber": 851
+  },
+  {
+  "name": "Jaideep Guntupalli",
```
Comment (krishguptadev) [inline data.json:1623]:
```suggestion
    "name": "Jaideep Guntupalli",
```

File: data.json [hunk 2]
```diff
@@ -1618,5 +1618,11 @@
...
+  "githubUsername": "JaideepGuntupalli",
```
Comment (krishguptadev) [inline data.json:1624]:
```suggestion
    "githubUsername": "JaideepGuntupalli",
```

File: data.json [hunk 3]
```diff
@@ -1618,5 +1618,11 @@
...
+  "imageUrl": "https://user-images.githubusercontent.com/63718527/142635090-b0065905-edad-4d60-81d1-206ddf3093a4.png",
```
Comment (krishguptadev) [inline data.json:1625]:
```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/63718527/142635090-b0065905-edad-4d60-81d1-206ddf3093a4.png",
```

File: data.json [hunk 4]
```diff
@@ -1618,5 +1618,11 @@
...
+  "issueNumber": 853
```
Comment (krishguptadev) [inline data.json:1626]:
```suggestion
    "issueNumber": 853
```

**Review by krishguptadev — CHANGES_REQUESTED** [review]

**Review by krishguptadev — APPROVED** [review]

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**